model/mapgeneration/src/main/kotlin/de/gleex/pltcmd/model/mapgeneration/mapgenerators/intermediate/MountainTopHeightMapper.kt

```diff
@@ -79,20 +78,33 @@ class MountainTopHeightMapper(override val rand: Random, override val context: G
         var tries = 0
         while (pickedAreas.size < mountainTopAreasToFind && tries < 1000) {
             val candidate = mainCoordinates.random(rand)
-            if(candidate.toCoordinate() in area) {
+            if (candidate.toCoordinate() in area) {
                 pickedAreas.add(candidate)
             }
             tries++
         }
 
         val pickedLocations = mutableSetOf<Coordinate>()
         pickedAreas.forEach { mainCoordinate ->
-            for(i in 0 until mountainTopsPerMainCoordinate)
-                pickedLocations.add(
-                        mainCoordinate.toCoordinate()
-                                .movedBy(rand.nextInt(100), rand.nextInt(100)))
+            for (i in 0 until mountainTopsPerMainCoordinate) {
```

Comment:
`i` is unuesd so it could just be 
```suggestion
            repeat (mountainTopsPerMainCoordinate) {
```